src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl

```diff
@@ -955,6 +955,15 @@ def _collect_bundle_info(target):
 
     return None
 
+def _has_dsym(target):
+    """Returns True if the given target provides dSYM, otherwise False."""
+    if apple_common.AppleDebugOutputs in target:
+        debug_outputs_provider = target[apple_common.AppleDebugOutputs]
+        outputs_map = debug_outputs_provider.outputs_map
```

Comment:
This was broken in f98f8e7cb when I swapped off of the objc fragment. I think it makes sense to use this instead, but we should iterate through the dictionaries in the outputs_map to see if any of them have "dsym_binary" set. Would also be good to add this to the E2E test that we have but I can handle that, not sure if it runs externally.

Reply:
Fixed. What I'm actually seeing in our builds is that outputs_map will be an empty dictionary when we're not building with dSYMs, though in that case, it just iterates through an empty dictionary so it seems fine.

Reply:
I think that may depend on what other debug outputs have been requested: https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/rules/objc/AppleBinary.java;l=262;drc=59f19ebc5187861ed56cd4fdd860e3a78ce4502b although I'm not sure if that codepath is still hit. But thanks LGTM